index.html

```diff
@@ -1,29 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap');
+    </style>
+    <link rel="stylesheet" href="styles.css">
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
-    <title>Document</title>
+    <title>Mobile Portfolio</title>
 </head>
 <body>
-    <header>
-        <ul class="nav-bar">
-            <li><a href="#">Porfolio</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">info</a></li>
-        </ul>
-    </header>
-    <main class="main-content">
-        <h1 class="heading">Hey there I,m Bunbee.<br>I'm a software developer</h1>
-        <p>I can help you build a product, feature or website. Look thrrough some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
-        <ul class="footer">
-            <li><a href="https://github.com/mutinhiri">github</a></li>
-            <li><a href="www.linkedin/mutinhiri">Linkedin</a></li>
-            <li><a href="twitter.com/bunbee">Twitter</a></li>
-        </ul>
-    </main>
-    
-</body>
-</html>
+    <section>
+        <nav class="header">
+            <a class="logo" href="#">Bunbee</a>
```

Comment:
- [ ] Since following the design is crucial during this module, please update the logo to look like the original design

| Yours | Original |
|--|--|
|![image](https://user-images.githubusercontent.com/11758151/130263877-5862db89-abf4-423f-b30c-0dcf1b109923.png)|![image](https://user-images.githubusercontent.com/11758151/130263929-1195ab0c-a0e5-4a7f-883e-7d7b0348acb2.png)|